scriptmodules/emulators/mupen64plus.sh

```diff
@@ -173,16 +173,14 @@ function install_mupen64plus() {
 function configure_mupen64plus() {
     local res
     local resolutions=("320x240" "640x480")
-    isPlatform "kms" && resolutions=("%XRES%x%YRES%")
+    isPlatform "kms" && res="%XRES%x%YRES%"
 
     if isPlatform "rpi"; then
-        # mesa needs to run at full screen as it doesn't benefit from our SDL scaling hint
-        if isPlatform "mesa"; then
-            res="${resolutions[0]}"
-            addEmulator 1 "${md_id}-GLideN64" "n64" "$md_inst/bin/mupen64plus.sh mupen64plus-video-GLideN64 %ROM% $res 0 --set Video-GLideN64[UseNativeResolutionFactor]\=1"
+        # kms needs to run at full screen as it doesn't benefit from our SDL scaling hint
+        if isPlatform "kms"; then
+            addEmulator 0 "${md_id}-GLideN64" "n64" "$md_inst/bin/mupen64plus.sh mupen64plus-video-GLideN64 %ROM% $res 0 --set Video-GLideN64[UseNativeResolutionFactor]\=1"
```

Comment:
I think you want this one with `addEmulator 1` as it’s the best default for kms

Reply:
I just forgot to make that default also.

Reply:
I've done that now.

Reply:
in https://github.com/RetroPie/RetroPie-Setup/issues/2961 i suggested mupen64plus standalone stays as default. it should be more performant/compatible as i think the libretro version isn't threaded yet, and may have some bugs that standalone does not.

Reply:
I'm not convinced. I may switch it.